src/main/java/slimeknights/tconstruct/library/client/RenderUtil.java.rej

```diff
@@ -1,16 +0,0 @@
-diff a/src/main/java/slimeknights/tconstruct/library/client/RenderUtil.java b/src/main/java/slimeknights/tconstruct/library/client/RenderUtil.java	(rejected hunks)
```

Comment:
Yep, accidently committed in the seared furnace commit. I was just deleting it here.